lib/buffer.js

```diff
@@ -270,6 +270,11 @@ function fromObject(obj) {
       }
       return fromArrayLike(obj);
     }
+    if (typeof SharedArrayBuffer !== 'undefined') {
```

Comment:
I would say yes. I think just changing the `if (isArrayBuffer(value))` to `if (isArrayBuffer(value) || isSharedArrayBuffer(value))` in `Buffer.from()` should work.